code/obj/item/cigarette.dm

```diff
@@ -258,7 +258,10 @@
 
 			var/mob/living/carbon/human/human_target = target
 			if (human_target && rand(1,5) == 1)
-				SPAWN(0) target.emote("cough")
+				SPAWN(0)
```

Comment:
```suggestion

```
This spawn is doing nothing here its best to just remove it. change the rand() stuff above to prob(20) as well.